tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 recreate = true
 skipsdist = true
-envlist = py{39,310,311,312}
+envlist = py{310,311,312}
```

Comment:
should we also add 313?

Reply:
Your call. If we test it and want to support it, I say yes. @jni

Reply:
Added it.